cmd/influxd/run/config_command.go

```diff
@@ -74,6 +74,7 @@ func (cmd *PrintConfigCommand) parseConfig(path string) (*Config, error) {
 	if _, err := toml.DecodeFile(path, &config); err != nil {
 		return nil, err
 	}
+    config.InitTableAttrs()
```

Comment:
@otoolep 
The question (your question :) ) is: Is this line needed ???
My answer: I don't think, But I'm not sure. Do you have any idea ? Thanks !